WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormEventLogger.swift

```diff
@@ -31,4 +31,12 @@ struct ProductFormEventLogger: ProductFormEventLoggerProtocol {
     func logSubscriptionsTapped() {
         ServiceLocator.analytics.track(event: .ProductDetail.subscriptionsTapped())
     }
+
+    func logSubscriptionsFreeTrialTapped() {
+        ServiceLocator.analytics.track(.productDetailsViewSubscriptionFreeTrialTapped)
+    }
+
+    func logSubscriptionsExpirationDateTapped() {
+        ServiceLocator.analytics.track(.productDetailsViewSubscriptionExpirationDateTapped)
```

Comment:
Nit: Consider following the pattern used in `func logSubscriptionsTapped()` above which calls `func subscriptionsTapped()` to get the event.

Reply:
Handled in https://github.com/woocommerce/woocommerce-ios/commit/20e2f8a8d73d989bd6928e576cc3ace90b59754a